### ISSUE TITLE
Filepicker to save configuration does not open on Linux

### ISSUE THREAD

**Issue by rgaudin** [kiwix/kiwix-hotspot#173]
In GitLab by @Kelson42 on Apr 26, 2018, 05:12

This is what is written on the console
```
Traceback (most recent call last):
  File "gui.py", line 421, in activate_menu_config
  File "gi/module.py", line 320, in __getattr__
  File "gi/module.py", line 139, in __getattr__
AttributeError: 'gi.repository.Gtk' object has no attribute 'FileChooserNative'

```

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Apr 27, 2018, 15:02

mentioned in commit d439a4b4dcdd5118f93dcbd7d5ff2168dff0ea06

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Apr 27, 2018, 15:03

mentioned in merge request !39

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Apr 27, 2018, 15:07

mentioned in commit 5166d49b23f8f12355771bbd097a408d919e2e72

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Apr 27, 2018, 15:14

mentioned in commit 6159aa80d0b39fe914986a54ccf1610a07abf1ac

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Apr 27, 2018, 15:18

mentioned in merge request !40

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on May 3, 2018, 16:35

mentioned in commit 7d50ef515d7e6c706c092b6673a89984244e8c39

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on May 3, 2018, 16:35

closed via commit 6159aa80d0b39fe914986a54ccf1610a07abf1ac

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on May 3, 2018, 16:35

closed via merge request !40
